pkg/sdn/plugin/controller.go

```diff
@@ -47,6 +47,33 @@ func getPluginVersion(multitenant bool) []string {
 	return []string{"00", version}
 }
 
+func (plugin *OsdnNode) getLocalSubnet() (string, error) {
+	// timeout: 30 secs
+	retries := 60
+	retryInterval := 500 * time.Millisecond
+
+	var err error
+	var subnet *osapi.HostSubnet
+	// Try every retryInterval and bail-out if it exceeds max retries
+	for i := 0; i < retries; i++ {
```

Comment:
Yeah, clayton pushed back on that for the CNI stuff too.  See https://github.com/openshift/origin/pull/9981/files#diff-6357e2d44bec4f49542401c788e87f51R426 for an example.